### PR TITLE
RSDK-7436: remove analog and digital statuses from python sdk

### PR DESCRIPTION
**Pull request by martha-johnston**

### CODE REVIEW

**Review by purplenicole730 — APPROVED** [review]

**Comment by martha-johnston** [comment]
hmm did not expect so many conflicts in the proto files... would it be easier for me to revert that commit and pull in the other automated PR that builds the protos?

**Review by purplenicole730 — COMMENTED** [review]
I'll look over this again once this PR is ready. (tests passing, no more commits)

**Comment by njooma** [comment]
@martha-johnston pull from main and then rerun `make buf`

File: src/viam/components/board/__init__.py [hunk 1]
```diff
@@ -19,12 +18,12 @@ async def create_status(component: Board) -> Status:
     for x in analog_names:
         analog = await component.analog_reader_by_name(x)
         read = await analog.read()
-        analogs[x] = AnalogStatus(value=read)
+        analogs[x] = int(read)
```
Comment (njooma) [inline src/viam/components/board/__init__.py:21]: Do you need this `int` conversion?

**Review by njooma — COMMENTED** [review]
Looks good. Just one question and one request: Can you run `make format` please?

File: src/viam/components/board/__init__.py (hunk 1 above)
Comment (martha-johnston) [reply src/viam/components/board/__init__.py]: I guess not just removed it

**Review by purplenicole730 — APPROVED** [review]

**Pull request merged**